compose/compose.go

```diff
@@ -74,6 +74,7 @@ func Compose(config *Config, storage interface{}, strategy interface{}, hasher f
 		ClientAuthenticationStrategy: config.GetClientAuthenticationStrategy(),
 		ResponseModeHandlerExtension: config.ResponseModeHandlerExtension,
 		MessageCatalog:               config.MessageCatalog,
+		FormPostHTMLTemplate:         config.FormPostHTMLTemplate,
```

Comment:
I think it is being used here?

https://github.com/ory/fosite/blob/cf02af977681fd667b33f8e131891f6746d0b9da/fosite.go#L114

Reply:
You're right, I thought that was also an addition, but it's not. Thank you, LGTM!